### PR TITLE
Add object-curly-spacing rule to eslint config

### PR DESCRIPTION
**Pull request by 102**

### CODE REVIEW

File: frontend/blocks/array/array_field.js [hunk 1]
```diff
@@ -29,32 +29,34 @@ export default class ArrayField extends Container {
...
-        Object.assign(config.actions.label, {
-          cls: config.actions.label.cls || 'col-sm-3'
+    const finalConfig = Object.assign({}, config, {
+      items: [{
+        block: 'fieldset',
+        id: fieldsetId,
+        label: config.itemLabel,
+        labelWidth: config.itemLabel ? 3 : 0,
+        layout: items.map(() => {
+          return { count: 3, width: [1, 8, 3] };
```
Comment (IgorZiegler) [inline frontend/blocks/array/array_field.js]: ```layout: items.map(() => ({ count: 3, width: [1, 8, 3] })```

File: frontend/blocks/array/array_field.js [hunk 2]
```diff
@@ -29,32 +29,34 @@ export default class ArrayField extends Container {
...
-      id: editorId,
-      items: [
-        Object.assign(config.actions.label, {
-          cls: config.actions.label.cls || 'col-sm-3'
+    const finalConfig = Object.assign({}, config, {
+      items: [{
+        block: 'fieldset',
+        id: fieldsetId,
+        label: config.itemLabel,
+        labelWidth: config.itemLabel ? 3 : 0,
```
Comment (IgorZiegler) [inline frontend/blocks/array/array_field.js]: Use const

File: frontend/blocks/array/array_field.js (hunk 1 above)
Comment (IgorZiegler) [inline frontend/blocks/array/array_field.js]: Use const

File: frontend/blocks/array/array_field.js (hunk 2 above)
Comment (102) [inline frontend/blocks/array/array_field.js]: >Add object-curly-spacing rule to eslint config

File: frontend/blocks/fieldset/fieldset.js [hunk 3]
```diff
@@ -15,8 +15,8 @@ export default class Fieldset extends Container {
     const items = config.items.map(item => Object.assign({}, item, { suppressLabel: true }));
     const finalConfig = Object.assign({}, config, { items });
 
-    finalConfig.layout = Array.isArray(config.layout) ? config.layout : [{count: config.items.length}];
-    finalConfig.layout = finalConfig.layout.map(rowConfig => Number.isInteger(rowConfig) ? {count: rowConfig} : rowConfig);
+    finalConfig.layout = Array.isArray(config.layout) ? config.layout : [{ count: config.items.length }];
+    finalConfig.layout = finalConfig.layout.map(rowConfig => Number.isInteger(rowConfig) ? { count: rowConfig } : rowConfig);
```
Comment (IgorZiegler) [inline frontend/blocks/fieldset/fieldset.js]: combined into a single string

Comment (102) [inline frontend/blocks/fieldset/fieldset.js]: >Add object-curly-spacing rule to eslint config

**Comment by 102** [comment]
@IgorZiegler I'm not sure that you intentionally closed this pr

**Pull request merged**